docs/Guides/Automate/TriggerAutomation.md

```diff
@@ -0,0 +1,60 @@
+# Trigger a prebuilt automation
+
+If you have setup automations in the Vendasta platform you can trigger them for a business. This enables you to set up a complex chain of actions but only need one api call. 
+
+## Setup:
+
+Create an access token with at least `business` scopes following the [Authorization guide](../../Authorization/Authorization.md).
+
+Create an automation in the Vendasta platform with a [Manual Trigger and turn it on](https://support.vendasta.com/hc/en-us/articles/4406958983703).
+
+## Step 1: Trigger the automation
+
+<!--
+type: tab
+title: Request
+-->
+
+You will need the ID of the automation you previously created as well as the business ID of the business location you would like to trigger that automation for. Add the access token to the headers and send it off. 
```

Comment:
We should add directions on how to get the IDs.

Reply:
Are there docs on how to get the business location IDs?
I'm currently working with Erika to figure out where to display the automation id, something like this

![Screen Shot 2021-09-08 at 12 18 47 PM](https://user-images.githubusercontent.com/6391814/132572198-844cc077-7562-495b-a54b-e21050baaab5.png)

Reply:
I like that for the automation ID. (It could use a bit more whitespace so it is not a wall of text.)
For the business location ID you can link to this guide 
https://docs.apigateway.co/docs/openapi-specs/branches/automation-runs-docs/docs/Guides/Accounts.md